syntax = "proto3";
service UserService {
 rpc CreateUser(CreateUserRequest) returns (UserResponse);
 rpc GetUser(GetUserRequest) returns (UserResponse);
}
message CreateUserRequest {
 string name = 1;
 string email = 2;
}
message UserResponse {
 string user_id = 1;
 string name = 2;
 string email = 3;
}
message GetUserRequest {
 string user_id = 1;
}